dnf/package.py

```diff
@@ -225,9 +225,9 @@ def schemes_filter(url_list):
                 if schemes:
                     s = dnf.pycomp.urlparse.urlparse(url)[0]
                     if s in schemes:
-                        return url + self.location
+                        return os.path.join(url, self.location)
```

Comment:
@ignatenkobrain Please why not to use here os.path.join()? Please provide the issue.
I have against the following:
The library has different imports for python2 and 3,  urllib.urljoin or urllib.parse.urljoin is not used in dnf, and finally I don't like the behavior provided by that function - potential issue.